linux_os/guide/system/logging/journald/socket_systemd-journal-remote_disabled/rule.yml

```diff
@@ -0,0 +1,31 @@
+documentation_complete: true
+
```

Comment:
I think this rule deserves a prodtype. If nothing, then the service / package seems not to exist on RHEL 7. Please add desired ubuntu prodtypes, other products can be added later.